news/2017-09-21-osu-coffee-hour-bnqat.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "osu! Coffee Hour: Beatmap Nominators and the QAT"
+date: 2017-09-20 19:30:00 +0000
+---
+
+The osu! Coffee Hour's back for its third sitting, featuring everything **Beatmap Nominators (BN)** / **Quality Assurance Team (QAT)** and the overhaul of the whole system that's on the horizon. Get your questions answered, and see where the future of the game's beatmapping scene is heading! 
+
+[![](/wiki/shared/news/banners/coffee-hour.jpg)](http://itsalmo.st/#osucoffeehour-bnqattime)
+
+## This week's theme
+
+This week, we'll be going over the history of the **Beatmap Nominators** and the **Quality Assurance Team**, along with the systems that have accompanied them. Changes are on the horizon - where is this all going, and what can we expect from the future?
+
+To help me ([HappyStick](https://osu.ppy.sh/u/happystick)) figure it all out, we'll be joined by the recently-voted QAT leaders, [JBHyperion](https://osu.ppy.sh/u/JBHyperion) and [Okorin](https://osu.ppy.sh/u/okorin). [pishifat](https://osu.ppy.sh/u/pishifat) will also be joining us again, using his keen interest in *osu!*'s history to provide perspective as we look forward.
```

Comment:
eh, shouldn't you use user ids?
`[HappyStick](https://osu.ppy.sh/users/256802)`
`[JBHyperion](https://osu.ppy.sh/users/4879508)`
`[Okorin](https://osu.ppy.sh/users/1623405)`
`[pishifat](https://osu.ppy.sh/users/3178418)`